apps/fabric-website/src/pages/Styles/Fabric7Page/Fabric7Page.tsx

```diff
@@ -5,7 +5,7 @@ export const Fabric7Page: React.FunctionComponent<IStylesPageProps> = props => {
   return (
     <StylesAreaPage
       {...props}
-      title="Fabric 7 Updates"
+      title="Fluent UI 7 Updates"
```

Comment:
In this context, I'm not sure if this really ought to be Fluent UI (same in main files for this page)